app/assets/stylesheets/provider/admin/cms/_templates.scss

```diff
@@ -35,31 +35,31 @@
   }
 }
 
-#cms-template-editor-tabs {
```

Comment:
Not needed, because `ul.ui-tabs-nav` provide the required styles.
This is the element:
<img width="963" height="322" alt="image" src="https://github.com/user-attachments/assets/6fbc8948-8271-4ae8-9b95-e516099300a6" />